index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    <nav class="menu">
+        <a href="#" class="logo">MIRNA</a>
+        <img src="./image/Enabled.png" alt="hamburger icon" class="hamburger_icon">
+    </nav>
+    <section class="header">
+        <h1 class="header_title">I'm Mirna, welcome to my portfolio!</h1>
+        <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <a href="#" class="final_msj">LET'S CONNECT</a>
+        <ul class="header_list">
+            <li><a href="#" class="header_icon"><img src="./image/twitter_icon.png" alt="twitter_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/Linkedin icon.png" alt="linkedin_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/m_icon.png" alt="m_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/github_icon.png" alt="github"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/V_icon.png" alt="V_icon"></a></li>
+        </ul>
+    </section>  
+    <section class="works">
+        <div class="works_card">
+            <img class="works_img" src="./image/works_1.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
```

Comment:
- [ ] I would suggest you please use the same background provided by microverse. See the image below for reference
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/192126381-c1ca8ae2-329e-4ea7-b9d9-43e007223e73.png)
### You design
![yourrrrrrrr](https://user-images.githubusercontent.com/81727988/192126388-cfcdecca-3280-4a00-afd5-e0ab7cdb1f3e.png)